fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -744,13 +744,22 @@ module Package =
 /// this includes Date::now and Int::random.
 /// </remarks>
 type Previewable =
-  /// Do not need to be saved, can be recalculated in JS
+  /// Can be recalculated in the editor, so we don't need to save results.
+  /// e.g. `Date::add`
   | Pure
 
-  /// Save their results. We can preview these safely
+  /// Can either:
+  /// - only be run on the server but is actually pure (e.g. `Crypto::md5`)
+  ///   Generally, these are functions that aren't compilable to WASM.
+  /// - be safely run without side effects but is not pure (e.g. `Date::now`)
+  ///
+  /// We should save the results.
   | ImpurePreviewable
```

Comment:
Seems I steered you wrong here as my understanding was off.

There are 5 functions currently marked ImpurePreviewable - those are all crypto functions, which are technically pure and can be moved over to Pure once the ocaml is turned off.

Then there are functions which can safely be run but not pure, but none of them are marked as ImpurePreviewable.

To be usable in the editor/wasm, a function must be both Pure and shipped as part of the Wasm build (eg excludes BackendOnlyStdLib).

Reply:
My current impression is that `ImpurePreviewable` might fall off entirely? In any case, going back-and-forth on this so will vastly simplify comment here to not spend too much time on it

Reply:
"Can be safely previewable, and we should save the results." feels pretty safe.

Reply:
OK I gave the wording on these another go; feels better to me,  anyway :)